DESCRIPTION

```diff
@@ -29,8 +29,7 @@ Suggests: knitr,
     rgeos,
     sf,
     sp,
-    tidyr,
-    tmap
+    tidyr
```

Comment:
I don't have an issue with removing `tmap`, but we'll have to remove it from the vignettes as well. I guess the plan is to replace it with `ggplot2` code?